app/src/main/java/com/github/polypoly/app/menu/ProfileModifyingActivity.kt

```diff
@@ -104,45 +90,144 @@ class ProfileModifyingActivity : ComponentActivity() {
         }
     }
 
+    /**
+     * Permit to select the trophies the user want to display on his/her profile
+     * @param trophiesDisplay trophies currently displayed on the user profile
+     * @param user the user to whom the profile belongs
+     */
+    @Composable
+    fun TrophiesSelection(trophiesDisplay: MutableList<Int>, user: User) {
+        Text("Select displayed trophies", style = MaterialTheme.typography.h5,
+            textAlign = TextAlign.Center)
+        Spacer(modifier = Modifier.height(20.dp))
+        TrophiesView(
+            callBack = { idx ->
+                if(!trophiesDisplay.contains(idx) && user.hasTrophy(idx)) {
+                    if (trophiesDisplay.size >= 3) trophiesDisplay.removeAt(0)
+                    trophiesDisplay.add(idx)
+                }
+            },
+            maxSelected = 3,
```

Comment:
3 here could be turned into a variable to keep consistency with the `trophiesDisplay.size >= 3` above